.github/actions/check_cirrus_cron/cron_failures.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# Intended to be executed from a github action workflow step.
+# Outputs the Cirrus cron names and IDs of any failed builds
+
+err() {
+    # Ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions
+    echo "::error file=${BASH_SOURCE[0]},line=${BASH_LINENO[0]}::${1:-No error message given}"
+    exit 1
+}
+
+if [[ -z "$GITHUB_REPOSITORY" ]]; then
+    err "Expecting \$GITHUB_REPOSITORY value to not be empty"
+elif [[ ! -w "$GITHUB_ENV" ]]; then
```

Comment:
Meant to ask earlier: what's this all about? `GITHUB_ENV` is not used anywhere else in this script, nor this PR, nor anywhere at all in the podman repo AFAICT

Reply:
oh thanks for asking.  This is from an earlier incarnation...Appending `<name>=<value>` pairs to that file causes subsequent steps env. vars. to be set.  The script use to pass data this way but I found dumping into the `name_id.csv` file to be easier.  I'll strip this out...